Neos.Flow/Classes/Mvc/DispatchComponent.php

```diff
@@ -95,7 +97,7 @@ public function handle(ComponentContext $componentContext)
     protected function prepareActionRequest(ComponentContext $componentContext)
     {
         $httpRequest = $componentContext->getHttpRequest();
-        $arguments = $httpRequest->getArguments();
+        $arguments = $httpRequest->getQueryParams();
```

Comment:
This should be `getAttribute(Http\Request::ATTRIBUTE_UNIFIED_ARGUMENTS)`

Reply:
I am currently not fully convinced this is a good idea, I created a little helper and just merge the arguemnts wherever I needed it. TBD...

Reply:
That's possible too. Wanted to avoid throwing that Helper around, but it's correct that it's probably less error prone, since it works with changed arguments somewhere inside the http component chain/middleware stack.